src/api/status/public/js/build-log/build-header.js

```diff
@@ -11,17 +12,19 @@ const buildStarted = document.getElementById('build-started');
 const buildDuration = document.getElementById('build-duration');
 const buildPrevious = document.getElementById('previous-build');
 const buildLoadingAnimation = document.getElementById('build-animation');
+const buildCommitMessage = document.getElementById('build-commit-message');
 
 function renderBuildTimeInfo(startedDate, stoppedDate = new Date()) {
   const duration = new Date(stoppedDate).getTime() - new Date(startedDate).getTime();
   buildDuration.innerText = ms(duration);
   buildStarted.innerText = new Date(startedDate).toUTCString();
 }
 
-function renderSender(sender) {
+function renderSender(sender, headCommit) {
```

Comment:
Since all you're using here is the `message`, why don't you call this `commitMessage` and do `renderSender(sender, head_commit.message)` below.